### ISSUE TITLE
Analyse comments made on 'places of worship' on the site to improve commenting guidelines and our moderation practices

### ISSUE THREAD

**Issue by katska** [openaustralia/planningalerts#1477]
What do we learn if we look at all the comments that have been made in response to 'place of worship' on PlanningAlerts?

## Why?
This issue initiated from noticing over time that these come up in our 'abuse reports'.  Someone wrote to us reporting a comment:

> there was no way at all any racism or disgrace against other race in other person comment so why do they have to try and create an argument and hate where is not needed"
> 
> The original comment was written by James who said:
> I have no concerns with this application. I wonder if this was a christian church (to compliment the EXISTING christian church in this street) would there be the same level of angst about "traffic".
> I travel on Watson road numerous times a day throughout the day. Traffic is no better or worse than it was since the Bunnings went in.
> The underlying discriminatory tone in these complaints is concerning, especially for a multicultural area like Padstow/Bankstown.

The comment stayed with me. So, last night, while I reflected on my recent experience with these comments, I searched for ['place of worship' site:planningalerts.org.au](https://www.google.com/search?client=safari&rls=en&ei=LViGX8XsHcGYmge-krXwDA&q=place+of+worship+site%3Aplanningalerts.org.au&oq=place+of+worship+site%3Aplanningalerts.org.au&gs_lcp=CgZwc3ktYWIQA1CUtQFYlLUBYJnVAWgBcAB4AIABowGIAaMBkgEDMC4xmAEAoAEBqgEHZ3dzLXdpesABAQ&sclient=psy-ab&ved=0ahUKEwiFt_D3-rLsAhVBjOYKHT5JDc4Q4dUDCAw&uact=5) and looked at the first few. Place of worship seems to have replaced "church" or "mosque" or "temple" in the planning parlance. This in itself could be an attempt to neutralise negative/racist reactions to specific religions. Yet I can't help but think that the people in an area would know who was behind the application, so I don't know if this doesn't create another strange dynamic in the process. The first few links I followed (in the later at night and brief first examination) contained a bunch of comments about bringing too much traffic mostly, and some statements about the nature of a place changing, that may or may not be innocent enough. People didn't refer to religion mostly.

I started to find myself assuming that these comments were from people fearful of change, and maybe those fears were also informed by racism if those people were not actually displaying overtly racist behaviour. It smacked of covert indirect behaviours.So I saw several of these. And then one specifically for a Christian church. The churchgoers dominated the many comments in this DA and were very clear to display information about which religion was coming to the site.
Before making too many judgements about anyone making comments or the kind of organising going on behind them, I figured it would be useful to initiate action to explore what we can see going on so far. 

## Why? 
Well to feel safe to be able to express ones freedom in a democracy we need to know that we are collectively safe to express ourselves. Freedom of religious expression is a fundamental right in Australia that I can remember from becoming a citizen material. https://www.aph.gov.au/Parliamentary_Business/Committees/Joint/Foreign_Affairs_Defence_and_Trade/Freedomofreligion/Interim_Report/section?id=committees%2Freportjnt%2F024110%2F25178

**Comment by mlandauer** [comment]
@katska any chance you could format the text a little better so it's more readable? I'm thinking using markdown headers and breaks between the paragraphs...

**Comment by katska** [comment]
good point. I'll look into it when I'm back on this task 👍 I invite you to have a first pass if you would like to flex your more instantly available markdown skills?

**Comment by katska** [comment]
Here's another application, this one brought to my attention by a couple of comment reports citing a comment (now hidden) as containing racist comments / cultural discrimination https://www.planningalerts.org.au/admin/applications/2286981

**Comment by katska** [comment]
I removed myself as the assignee because I'm not actively following up on this right now.

**Comment by JoannaHill** [comment]
Stil excellent research idea informing better moderation.
